docs/_posts/2017-02-17-bulkoad-ingest-sst-file.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: Bulkoading by ingesting external SST files
+layout: post
+author: IslamAbdelRahman
+category: blog
+---
+
+## Introduction
+
```

Comment:
It would be nice that in the very beginning, we summarize what this blog post is about, to capture those who are interested. Bulk load is a well-known concept. So maybe we can throw out this idea before introducing the difference from the normal write path.